### PR TITLE
Bug 1119013 - [Messages][Subject] Focus is not given to the subject area of a message when adding a subject to the message. r=schung

### PR DESCRIPTION
**Pull request by azasypkin**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[azasypkin Aleh Zasypkin (azasypkin)](https://github.com/azasypkin) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=998df15faeb3)

**Comment by try-server-hook** [comment]
[azasypkin Aleh Zasypkin (azasypkin)](https://github.com/azasypkin) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=b6b7eb2d588c)

**Comment by try-server-hook** [comment]
[azasypkin Aleh Zasypkin (azasypkin)](https://github.com/azasypkin) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=5132b9803f81)

File: apps/sms/test/marionette/lib/messages.js [hunk 1]
```diff
@@ -237,6 +237,8 @@
               break;
             }
           }
+
+          client.helper.waitForElementToDisappear(menuElement);
```
Comment (azasypkin) [inline apps/sms/test/marionette/lib/messages.js]: This is to be sure that menu is disappeared for sure :)

File: shared/js/option_menu.js [hunk 2]
```diff
@@ -121,6 +121,7 @@ var OptionMenu = function(options) {
   // For each option, we append the item and listener
   items.forEach(function renderOption(item) {
     var button = document.createElement('button');
+    button.type = 'button';
```
Comment (azasypkin) [inline shared/js/option_menu.js]: Will need "shared" peer review for this change as well

**Pull request merged**